test/StressTests/Databases.cs

```diff
@@ -18,7 +18,7 @@ namespace StressTests
 {
     public class Databases : RavenTestBase
     {
-        [NightlyBuildTheory]
+        [NightlyBuildTheory64]
```

Comment:
Let's be consistent. Either go with `<name>32|64` or `<name>32Bit|64Bit` for all attribute names.